System/Desk/Desk.ts

```diff
@@ -243,11 +233,10 @@ export class Desk {
         this.lastDragApp = null;
         this.currentDragApp = null;
 
-        // TODO: use a fat arrow function instead of .bind(this)
         this.dragEvent = new DeskEvent(
             document.body,
             "mousemove",
-            (evt: { clientX: number; clientY: number }) => {
+            (evt: MouseEvent) => {
```

Comment:
The `MouseEvent` type annotation shouldn't be necessary

Reply:
actually, I prefer to explicitly say the type all the time, but we can talk about the convention in the meetings